components/partial/MDXStyleWrapper.tsx

```diff
@@ -100,7 +100,7 @@ const MDXStyleWrapper = ({ children }: { children: any }) => (
         }
       }
 
-      > div {
+      > div:not(.mdx-style-override) {
         margin-top: 29px;
         margin-bottom: 29px;
       }
```

Comment:
These margins were to add whitespace between adjacent components, but had the side effect of adding that margin to the banner components on the guide/concept index pages as well. This removes that gap again

Reply:
Oops! Good catch!